src/pages/test-report/test-report.ts

```diff
@@ -111,7 +110,9 @@ export class TestReportPage extends BasePageComponent {
       isRemoveFaultMode$.pipe(map(result => this.isRemoveFaultMode = result)),
       isSeriousMode$.pipe(map(result => this.isSeriousMode = result)),
       isDangerousMode$.pipe(map(result => this.isDangerousMode = result)),
-      manoeuvres$.pipe(map(manoeuvres => this.manoeuvresCompleted = Object.values(manoeuvres)[0])),
+
+      // TODO: [MES-2046] Not correct, controlled stop is not part of the manoeuvres
```

Comment:
Just for my benefit what is there to do here exactly?

Reply:
Nothing I forgot to remove the TODO - It's one szabi left for me yesterday